src/devices/bus/isa/mda.cpp

```diff
@@ -965,3 +975,514 @@ MC6845_UPDATE_ROW( isa8_ec1840_0002_device::crtc_update_row )
+
+  This board is normaly used with an Ericsson monitor due to the non standard connector.
+  Trivia: https://www.pinterest.se/pin/203084264425177097/
+ */
+
+#define EPC_MDA_SCREEN "epc_mda_screen"
+
+#if 0
+static GFXDECODE_START( pcepc )
+	GFXDECODE_ENTRY( "gfx1", 0x0000, pc_16_charlayout, 1, 1 )
+GFXDECODE_END
+#endif
+
+ROM_START( epc )
+	ROM_REGION(0x2000,"chargen", 0)
+	ROM_LOAD("8363_65_14_80_cg_50821_a64.bin",  0x00000, 0x2000, CRC(be709786) SHA1(38ab26224bbe66bbe2bb2ccac29b41cbf78bdbf8))
+	//ROM_LOAD("10_40_vp_402_28_ic_24_a19.bin",  0x00000, 0x2000, CRC(2aa53b92) SHA1(87051a037249eb631d7d2191bc0e925125c60f39))
+ROM_END
+
+//**************************************************************************
+//  GLOBAL VARIABLES
+//**************************************************************************
+DEFINE_DEVICE_TYPE(ISA8_EPC_MDA, isa8_epc_mda_device, "isa_epc_mda", "Ericsson PC Monochrome HR Graphics Board 1070")
+
+
+//-------------------------------------------------
+//  device_add_mconfig - add device configuration
+//-------------------------------------------------
+/* There are two crystals on the board: 19.170Mhz and 17.040MHz  TODO: verify use */
+/* Text modes uses 720x400 base resolution and the Graphics modes 320/640x200/400 */
+/* This matches the difference between the crystals so we assume this for now     */
+void isa8_epc_mda_device::device_add_mconfig(machine_config &config)
+{
+	screen_device &screen(SCREEN(config, EPC_MDA_SCREEN, SCREEN_TYPE_RASTER));
+	screen.set_raw(XTAL(19'170'000) / 4, 720, 0, 720, 400, 0, 400);
+	//screen.set_screen_update(MC6845_NAME, FUNC(h46505_device::screen_update));
+	screen.set_screen_update(MC6845_NAME, FUNC(mc6845_device::screen_update));
+
+	PALETTE(config, m_palette).set_entries(4);
+
+	HD6845S(config, m_crtc, XTAL(19'170'000) / 16); // clock and divider are guesswork
+	m_crtc->set_screen(EPC_MDA_SCREEN);
+	m_crtc->set_show_border_area(false);
+	m_crtc->set_char_width(8);
+
+	m_crtc->set_update_row_callback(FUNC(isa8_epc_mda_device::crtc_update_row), this);
+	m_crtc->out_hsync_callback().set(FUNC(isa8_epc_mda_device::hsync_changed));
+	m_crtc->out_vsync_callback().set(FUNC(isa8_epc_mda_device::vsync_changed));
+
+	//MCFG_GFXDECODE_ADD("gfxdecode", "palette", pcepc)
+}
+
+//-------------------------------------------------
+//  rom_region - device-specific ROM region
+//-------------------------------------------------
+
+const tiny_rom_entry *isa8_epc_mda_device::device_rom_region() const
+{
+	return ROM_NAME( epc );
+}
+
+//**************************************************************************
+//  LIVE DEVICE
+//**************************************************************************
+
+//-------------------------------------------------
+//  isa8_epc_mda_device - constructor
+//-------------------------------------------------
+
+isa8_epc_mda_device::isa8_epc_mda_device(const machine_config &mconfig, const char *tag, device_t *owner, uint32_t clock) :
```

Comment:
Be consistent with the rest of the file – this file seems to have commas at the end of lines in member initialiser lists.

Reply:
fixed in the upcoming commit